optimize_workspace.sh

```diff
@@ -1,19 +1,33 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
-command -v shellcheck > /dev/null && shellcheck "$0"
+command -v shellcheck >/dev/null && shellcheck "$0"
 
 export PATH="$PATH:/root/.cargo/bin"
 
 rustup toolchain list
 cargo --version
 
-optimize_workspace.py
+# Build artifacts
+echo -n "Building artifacts in workspace..."
+/usr/local/bin/build_workspace.py
+echo "done."
 
-# Postprocess artifacts
+echo -n "Optimizing artifacts in workspace..."
+# Start clean
+rm -rf ./artifacts
```

Comment:
This removes files from the host that might be unrelated to our build. This is not so nice. Let's collect all artifacts somewhere in the gues and then copy to `/code/artifacts` at the end

Reply:
Good point. I'll fix it.